synbiohub_adapter/upload_sbol/upload_sbol.py

```diff
@@ -88,12 +91,27 @@ def submit_collection(self, doc, collection_id, collection_version, collection_n
         doc.description = collection_description
         doc.version = collection_version
 
-        response = self.part_shop.submit(doc)
+        try:
+            response = self.part_shop.submit(doc)
 
-        if response == 'Submission id and version already in use':
-            raise DuplicateCollectionError(doc.displayId, doc.version)
-        else:
             print(response)
+        except RuntimeError as e:
```

Comment:
This is a very broad exception to catch. Is there a more narrow exception, specific to the situation at hand, that could be caught instead?

Reply:
I don't think there is a more narrow exception to catch, so I've added a comment describing the issue.